hal_st/stm32fxxx/SpiSlaveStmDma.cpp

```diff
@@ -59,6 +59,34 @@ namespace hal
         return rxCancelled || txCancelled;
     }
 
+    void SpiSlaveStmDma::EnableChipSelectInterrupt()
+    {
+        slaveSelectGpio.EnableInterrupt([this]()
+            {
+                if (HasObserver())
+                {
+#ifdef SPI_NSS_POLARITY_LOW
+                    if (!slaveSelectGpio.Get())
+#else
+                    if (slaveSelectGpio.Get())
+#endif
```

Comment:
SPI_NSS_POLARITY_LOW is just a value to initialize NSSPolarity (if available). It is not an indication that low or high priority is used; if you really want to support that, the value used in CR1->SSI should be used instead